roles/local_repo_setup/tasks/main.yml

```diff
@@ -0,0 +1,32 @@
+---
+# vim: set ft=ansible:
+#
+# This role sets up local repo of the booted deployment
+#
+- name: Fail if branch_name is undefined
+  fail:
+    msg: "branch_name is undefined"
+  when: branch_name is undefined
+
+- name: Get rpm-ostree status output
+  command: rpm-ostree status --json
+  register: ros
+
+- name: Convert rpm-ostree output to JSON
+  set_fact:
+    ros_json: "{{ ros.stdout|from_json }}"
+
+- name: Set current checksum
+  set_fact:
+    checksum: "{{ ros_json['deployments'][0]['checksum'] }}"
+  when: ros_json['deployments'][0] is defined and
+        ros_json['deployments'][0]['booted']
+
+- name: Set current checksum
+  set_fact:
+    checksum: "{{ ros_json['deployments'][1]['checksum'] }}"
+  when: ros_json['deployments'][1] is defined and
+        ros_json['deployments'][1]['booted']
+
+- name: Create local repo
```

Comment:
@cgwalters I think my terminology is slightly off on repo and branch.  I will rename the roles to use branch instead of repo.